_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+* Fully consolidate tfjob-operator to training-operator ([#1850](https://github.com/kubeflow/training-operator/pull/1850))
+
+
+## Kserve
+
+Kubeflow v1.8 introduces KServe 0.11. In this release, we've introduced Large Language Model (LLM) runtimes, made enhancements to the KServe control plane, and updated the Python SDK with revised support for the Open Inference Protocol, as well as improved dependency management using [poetry](https://python-poetry.org/docs/). A selection of these features and bug fixes include:
+
+* TorchServe 0.8.0 for LLM support ([#2980](https://github.com/kserve/kserve/pull/2980))
+* Implement v2/open inference endpoints for kserve python runtimes([2655](https://github.com/kserve/kserve/pull/2655))
+* Adding an end-to-end example for deploying a large language model on KServe ([2836](https://github.com/kserve/kserve/pull/2836))
+
+More information on the KServe 0.11 release can be found in the [release notes](https://github.com/kserve/kserve/releases/tag/v0.11.0/)
+
+### Simplifying Storage for ML Workflows
+
+Kubeflow’s web apps enhancements expose valuable information to data scientists, which simplifies advanced workflows.  For example, Kubernetes storage management, especially for persistent volumes, is critical for rapid data science iteration but can require complex, manual configuration.  The newly introduced PVC Viewer makes storage management easy for end users and eliminates the need to learn low-level CLI commands.    Users can click and drag to add/remove files and file directories from president volumes.  The volumes can easily be attached to MLOps objects, like notebooks, which makes experimenting with different data sets faster.
+
+
+![Kubeflow user using pvc-viewer](/images/2023-10-23-kubeflow-1.8-release/pvc-viewer-demo.gif)
+
+
+### Support for ARM Machines 
+
+
+Kubeflow 1.8 brings forward a significant upgrade [(#7343)](https://github.com/kubeflow/kubeflow/pull/7342) by officially introducing support for ARM architectures. This move ensures that developers can leverage Kubeflow's powerful machine learning capabilities on a wider variety of devices and platforms, including the likes of Apple Silicon and a multitude of IoT devices. This expansion highlights Kubeflow's adaptability, ensuring seamless machine learning workflows across various hardware architectures.
+
+
+### Security - Rootless Kubeflow Updates and Fixes
+
+In our ongoing commitment to bolstering security in the project, version 1.8 brings both vital security fixes and innovative features. A notable endeavor by the Security has been the development of features that lay the groundwork for a [rootless Kubeflow #2455](https://github.com/kubeflow/manifests/pull/2455) in the future. v1.8 offers users the flexibility to run Kubeflow either with the optional istio-cni or without it. A selection of these features and bug fixes include:
```

Comment:
Security Team

Reply:
I discussed it with david and the latest version looks good now.

Reply:
There is a small typo:

```suggestion
In our ongoing commitment to bolstering security in the project, version 1.8 brings both vital security fixes and innovative features. A notable endeavor by the Security Team has been the development of features that lay the groundwork for a [rootless Kubeflow #2455](https://github.com/kubeflow/manifests/pull/2455) in the future. v1.8 offers users the flexibility to run Kubeflow either with the optional istio-cni or without it. A selection of these features and bug fixes include:
```